File: 1021-remove-outermost-parentheses/1021-remove-outermost-parentheses.js
/**
 * @param {string} s
 * @return {string}
 */
var removeOuterParentheses = function(s) {
      let res="";
     let count=0;
        for(let i=0;i<s.length;i++){
            if(s[i]=='(' && count==0)
                count++;
            else if(s[i]=='(' && count>=1){
                res+=s[i];
                count++;
            }   
            else if(s[i]==')' && count>1){
                res+=s[i];
                count--;
            }
            else if(s[i]==')' && count==1)
                count--;
        }
        return res;
};

File: 1021-remove-outermost-parentheses/1021-remove-outermost-parentheses.py
class Solution:
    def removeOuterParentheses(self, s: str) -> str:
        res=""
        count=0;
        length=len(s)
        for i in range(0,length):
            if s[i]=='(' and count==0:
                count=count+1
            elif s[i]=='(' and count>=1:
                res=res+s[i]
                count=count+1
            elif s[i]==')' and count>1:
                res=res+s[i]
                count=count-1;
            elif s[i]==')' and count==1:
                count=count-1;
            i=i+1
        return res;
        